.github/PROCESS_GUIDELINES.md

```diff
@@ -28,25 +26,25 @@ Here is a brief summary of the most important labels:
 
 - `New topic`: Entirely new documents
 - `Major update`: Significant new info: new section in existing topic, etc.
-- `Technical`: Changes to technical content/code/processes/naming conventions (any change to technical content)
+- `Minor update`: Changes to technical content/code/processes/naming conventions (any change to the information provided)
```

Comment:
So now the label is different MerchDocs isn't aligned with DevDocs on this? Do you do What's New? If so, that will have to be reconfigured to pull the right labels.

Reply:
We have been using this alternative label since the beginning  really. It makes more sense. Our Whats New config will have to different anyway based on repo structure, etc.